django_mongodb/compiler.py

```diff
@@ -83,7 +83,7 @@ def _make_result(self, entity, columns):
                 if column_alias is not None and column_alias != self.collection_name
                 else entity
             )
-            result.append(obj.get(name, col.field.get_default()))
+            result.append(obj.get(name))
```

Comment:
So, maybe is time to do `obj[name]` because None is a value, and returning it is a population ? or I am missing something?

Reply:
I think it's better to return `None` rather than fail with an exception here if a field doesn't exist in the document. For example, if a field is added to an existing model, we might not add that field to existing documents.

Reply:
🤔 , well so we are not managing migrations as the SQLs do, right?